packages/create-sitecore-jss/src/templates/react/src/components/styleguide/Styleguide-RouteFields.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { withSitecoreContext, Text } from '@sitecore-jss/sitecore-jss-react';
-import StyleguideSpecimen from '../Styleguide-Specimen';
+import StyleguideSpecimen from '../styleguide/Styleguide-Specimen';
```

Comment:
do we need to import Styleguide-Specimen through parent folder here instead of just './Styleguide-Specimen' ?